ruby/app/models/cart_product.rb

```diff
@@ -0,0 +1,4 @@
+class CartProduct < ActiveRecord::Base
```

Comment:
When you use `has_and_belongs_to_many` relation you don't need to create this model. Such a model creating in the case when you organize `has_many :through` relation